src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -133,9 +130,11 @@ std::string const& getFullVersionString ()
 {
     struct PrettyPrinter
     {
-        PrettyPrinter () : fullVersionString ("rippled-" + getVersionString ()){}
+        PrettyPrinter ()
+            : fullVersionString ("rippled-" + getVersionString ())
+        { }
```

Comment:
I think this can be simplified to just:
```
std::string const& getFullVersionString ()
{
    static std::string const value = "rippled-" + getVersionString ();
    return value;
}
```